packages/styling/src/styles/theme.ts

```diff
@@ -107,12 +107,20 @@ export function createTheme(theme: IPartialTheme, depComments: boolean = false):
     ...theme.semanticColors
   };
 
+  let defaultFontStyles: IFontStyles = DefaultFontStyles;
+  if (theme.defaultFontStyle) {
+    for (const fontStyle of Object.keys(DefaultFontStyles)) {
+      defaultFontStyles[fontStyle] = { ...defaultFontStyles[fontStyle], ...theme.defaultFontStyle };
+    }
+  }
+
   return {
     palette: newPalette,
     fonts: {
-      ...DefaultFontStyles,
+      ...defaultFontStyles,
       ...theme.fonts
     },
+    defaultFontStyle: theme.defaultFontStyle || DefaultFontStyles.medium,
```

Comment:
Since this function returns an object of type ITheme, it requires `defaultFontStyle`. I would make it optional, but I know that we discussed making everything on ITheme required a while back.

Reply:
Is this the exception? Should we make this an optional property of ITheme?

Reply:
I would be ok making it only a part of IPartialTheme.

Reply:
Basically `defaultFontStyle` is only used by `createTheme` to create the fonts. Readers of the theme will refer to `fonts.*` but will never need `theme.defaultFontStyle`.